code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -88,6 +88,7 @@
 	poison_per_bite = 3
 	var/atom/movable/cocoon_target
 	var/fed = 0
+	var/static/list/consumed_mobs = list() //mobs that have been consumed by nurse spiders to lay eggs

```

Comment:
Fox is right though this is holding references to things and they don't know about it so don't clear from that list when they're qdeleted.